File: ex04/ex04_03/ex04_03.py
import math

def main():
  oper = input("Enter +, -, *, /, %: ")
  num1 = int(input("Enter number 1: "))
  num2 = int(input("Enter number 2: "))

  match (oper):
    case '+':
      print(f"{num1} + {num2} = {num1 + num2}")
    case '-':
      print(f"{num1} - {num2} = {num1 - num2}")
    case '*':
      print(f"{num1} * {num2} = {num1 * num2}")
    case '/':
      print(f"{num1} / {num2} = {num1 / num2}")
    case '%':
      print(f"{num1} % {num2} = {math.floor(num1) % math.floor(num2)}")
    case _:
      print("No operation is performed")
  return 0

if __name__ == "__main__":
  main()

File: ex04/ex04_03/ex04_03.c
#include <stdio.h>

int main() {
  char oper;
  float num1, num2;
  printf("Enter +, -, *, /, %%: ");
  scanf("%c", &oper);
  printf("Enter number 1: ");
  scanf("%f", &num1);
  printf("Enter number 2: ");
  scanf("%f", &num2);

  switch (oper) {
    case '+':
      printf("%.2f + %.2f = %.2f\n", num1, num2, num1 + num2);
      break;
    case '-':
      printf("%.2f - %.2f = %.2f\n", num1, num2, num1 - num2);
      break;
    case '*':
      printf("%.2f * %.2f = %.2f\n", num1, num2, num1 * num2);
      break;
    case '/':
      printf("%.2f / %.2f = %.2f\n", num1, num2, num1 / num2);
      break;
    case '%':
      printf("%d %% %d = %d\n", (int) num1, (int) num2, (int) num1 % (int) num2);
      break;
    default:
      printf("No operation is performed");
  }
  return 0;
}